src/test/java/org/ehrbase/fhirbridge/fhir/patient/PatientIT.java

```diff
@@ -35,12 +43,23 @@ void createPatient() throws IOException {
         create("create-patient.json");
     }
 
+    @Test
+    void mappingPatient() throws IOException {
+        Patient patient = (Patient) super.testFileLoader.loadResource("create-patient.json");
+        PatientCompositionConverter patientCompositionConverterConverter = new PatientCompositionConverter();
+        GECCOPersonendatenComposition mappedGeccoPersonendatenComposition = patientCompositionConverterConverter.toComposition(patient);
+        Diff diff = compareCompositions(getJavers(), "paragon-GECCO-patient-mapping-output.json", mappedGeccoPersonendatenComposition);
+
+        assertEquals(0, diff.getChanges().size());
+    }
+
     @Test
     void createInvalid() throws IOException {
```

Comment:
Looks good so far, nevertheless you programmed two exception in your converter are both covered by this single invalid test ?

Reply:
Actually three exceptions and one test ;-) I now have one test for each exception.